src/core/logic.py

```diff
@@ -1820,3 +1821,46 @@ def get_active_proposal_form():
         active_form = models.ProposalForm.objects.filter(in_edit=False).first()
 
     return active_form
+
+
+def get_file_mimetype(file_path):
+    """
+
+    Args:
+        file_path (str): the path to the storage location of the file.
+
+    Returns:
+        str
+    """
+    mimetype_guess = mimetypes.guess_type(file_path)
+
+    try:
+        mimetype = mimetype_guess[0]
+        if not mimetype:
+            mimetype = 'unknown'
+    except IndexError:
+        mimetype = 'unknown'
+
+    return mimetype
+
+
+def get_file_content_dispostion(original_filename):
+    """
+
+    Args:
```

Comment:
Params not Args

Reply:
The skeleton docstrings are automatically generated by my IDE based on Google's standard.
Unfortunately I can't deviate from the standard just because it uses the wrong terminology.